media/js/freesound.js

```diff
@@ -154,3 +154,25 @@ function afterDownloadModal(show_modal_url, sound_name){
             openModal();
         });
 }
+function unsecureImageCheck(input) {
+    var unique = '#unsecure_img_check' + new Date().valueOf();
+    var div = $("<div>", {class: "unsecure_image_warning"});
+    div.append("<b>Warning</b>: you are adding an image from a non-secure location. Only images from secure locations (i.e. the URL starts with https://...) will be shown in Freesound.");
+    div.insertBefore(input);
+    div.hide();
+
+    function show_message_if_insecure(){
+        var txt = input.val();
+        var regular_expression = new RegExp('.*<img.+src=.http:.*', 'i');
```

Comment:
you don't need quotes around an attribute, so `<img src=http://foo>` should work too. This means the regex could better be: `'.*<img.+src=.?http:.*'`